locallibs/fix.py

```diff
@@ -119,3 +124,17 @@ def fix_other_things(framework_path, short_version):
     future'''
     return (ensure_current_version_link(framework_path, short_version) and
             fix_script_shebangs(framework_path, short_version))
+
+
+def fix_broken_signatures(files_relocatablized):
+    """
+    Unsign the binaries and libraries that were relocatablized to avoid
+    them having corrupted signatures.
+    """
+    for file in files_relocatablized:
+        print("Unsigning %s to avoid broken signature." % file)
```

Comment:
This strikes me as pretty noisy, but I can tweak it later to reduce the noise.

Reply:
Yeah, very agreed, I decided to print them out because seems like we also print out other actions on these files as well, so I was trying to make the loggings consistent with other places.